packages/flutter/lib/src/painting/_network_image_io.dart

```diff
@@ -122,3 +181,81 @@ class NetworkImage extends image_provider.ImageProvider<image_provider.NetworkIm
   @override
   String toString() => '$runtimeType("$url", scale: $scale)';
 }
+
+class _PendingLoadRequest {
+  _PendingLoadRequest(this.sendRequest, this.handleError);
+
+  Function sendRequest;
+  Function handleError;
+}
+
+
+class _DownloadRequest {
+  _DownloadRequest(this.sendPort, this.uri, this.headers, this.debugNetworkImageHttpClientProvider);
+
+  final SendPort sendPort;
+  final Uri uri;
+  final Map<String, String> headers;
+  final HttpClientProvider debugNetworkImageHttpClientProvider;
+}
+
+class _HttpClientIsolateParameters {
+  _HttpClientIsolateParameters(this.sendPort);
+
+  final SendPort sendPort;
```

Comment:
Consider calling this `mainIsolateSendPort` to be clear which isolate it's being used to communicate with.

Reply:
Done